shard.yml

```diff
@@ -1,5 +1,5 @@
 name: postgres_adapter
-version: 0.4.1
+version: 0.4.2
```

Comment:
not sure if I have to bump up the adapter version or just updating the deps versions below is enough